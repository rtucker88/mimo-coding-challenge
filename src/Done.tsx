import * as React from 'react';

const Done: React.StatelessComponent<{}> = () => {
    return (<div>Done</div>);
};

export default Done;