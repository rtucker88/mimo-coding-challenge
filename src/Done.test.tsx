import { mount } from 'enzyme';
import * as React from 'react';
import Done from './Done';

it('renders correctly', () => {
    const wrapper = mount(<Done />);
    expect(wrapper).toMatchSnapshot();
});
