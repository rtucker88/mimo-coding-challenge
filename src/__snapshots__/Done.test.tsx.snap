// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Done>
  <div>
    Done
  </div>
</Done>
`;
